app/modules/inventory/inventory.coffee

```diff
@@ -101,15 +102,22 @@ showItemsFromModels = (items)->
   switch items.length
     when 0 then app.execute 'show:error:missing'
     # redirect to the item
-    when 1 then showItemModal items.models[0]
+    when 1
+      item = items.models[0]
+      fallback = -> API.showUserInventory item.get('owner'), true
+      showItemModal item, fallback
     else showItemsList items
 
-showInventory = (options = {})->
+showInventory = (options)->
+  { user, group, section } = options
+  unless user? or group? or section?
```

Comment:
i guess this will break with shelves merging (and will have to be updated anytime a new category comes in). I understand early return are nice (but maybe there are other reasons) but why not trusting the inventoryLayout to figure out itself its options at initialisation instead ?

Reply:
that was probably convenient for development then, but doesn't cost much to remove: done in dd5aac42